src/hooks/jitPlugin.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Hook } from '@oclif/core';
+import { Prompter } from '@salesforce/sf-plugins-core';
+import { SfError } from '@salesforce/core';
+
+const hook: Hook<'jit_plugin_not_installed'> = async function (opts) {
+  const prompter = new Prompter();
+  const { confirmation } = await prompter.timedPrompt<{ confirmation: string }>([
```

Comment:
I'd like to see this be an info/warning instead of a prompt, so that this feature would work seamlessly in a CI environment.

We'd still be calling the current plugin install logic, which would cause a prompt for unsigned plugins, right?